runtimes/standalone/src/migrations/crowdloan_contributions.rs

```diff
@@ -0,0 +1,70 @@
+// KILT Blockchain – https://botlabs.org
+// Copyright (C) 2019-2021 BOTLabs GmbH
+
+// The KILT Blockchain is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// The KILT Blockchain is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+// If you feel like getting in touch with us, you can do so at info@botlabs.org
+
+use crate::Runtime;
+#[cfg(feature = "try-runtime")]
+use frame_support::traits::GetStorageVersion;
+use frame_support::traits::{OnRuntimeUpgrade, StorageVersion};
+use hex_literal::hex;
+use kilt_primitives::AccountId;
+
+// Same as Mashnet transfer account.
+pub const NEW_ADMIN_ACCOUNT: [u8; 32] = hex!("6a3c793cec9dbe330b349dc4eea6801090f5e71f53b1b41ad11afb4a313a282c");
+
+pub struct CrowdloanContributionsSetup;
+
+impl OnRuntimeUpgrade for CrowdloanContributionsSetup {
+	#[cfg(feature = "try-runtime")]
+	fn pre_upgrade() -> Result<(), &'static str> {
```

Comment:
I suppose we have separate migrations because of different admin accounts. There should still be a more elegant way. After re-checking the admin accounts here and realizing these are the corresponding transfer accounts, we should be able to do this quite simple:

S: Move migration to crowdloan pallet
* `pre_upgrade` is the same for all runtimes
*  `on_runtime_upgrade` and `post_upgrade` differ in the admin account, which can be retrieved by querying the storage for the current `transfer_account`.

Reply:
I was also going to implement the migration within the pallet, but @weichweich suggested to implement it at the runtime level. I had no strong opinion about it honestly, with the difference that if we do it at the pallet level, we do not know what is the admin level that **should** be set in the migration, while we know that in the runtime (even though we are not able to distinguish between different deployments of the same runtime, e.g., WILT and Spiritnet).

Reply:
If we move it into the pallet we could query the transfer_account. But then we would need to have the `kilt-launch` pallet as a dependency.

I would say, since we will remove the migrations rather soon, this should be fine.

Reply:
I would remove the migration from the standalone runtime. THere is no network that needs migrating.

Reply:
Yes it is just for quick prototyping and testing by spinning up a mashnet node, as the local parachain setup has some issues right now.